### ISSUE TITLE
Для OC 3

### ISSUE THREAD

**Issue by special-for** [KirilLoveVE/opencart2-exchange1c#44]
Переделал для 3й ки, но застрял на одном моменте...   запутался в в большом количестве кнопок... предлагаю: скину файли - посмотрите что-да как ?

**Comment by makarovda1984** [comment]
Да, присылайте посмотрим, как раз есть сейчас сайт на opencart3 18.02.2020, 00:08, "special-for" <notifications@github.com>:Переделал для 3й ки, но застрял на одном моменте... запутался в в большом количестве кнопок... предлагаю: скину файли - посмотрите что-да как ?—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.
